helm_deploy/values-dev.yaml

```diff
@@ -38,3 +38,4 @@ env:
 
 jobs:
   cronspec: "0 8,12,15 * * *"
+# jobs will run at 9am, 1pm and 4pm
```

Comment:
or maybe 8am, midday and 3pm in the winter? 😅

Reply:
yeah maybe.